### ISSUE TITLE
Support for React 19

### ISSUE THREAD

**Issue by dbelob** [Kukkimonsuta/inversify-react#20]
React 19 is released. Please add support for the new version of React to the library.

Content of `package.json` file:
```
"peerDependencies": {
  "inversify": "^5.0.5 || ^6.0.1",
  "react": "^16.3.0 || ^17.0.0 || ^18.0.0"
}
```
@Kukkimonsuta Thanks a lot for the excellent and useful library!

**Comment by dbelob** [comment]
@Kukkimonsuta Please see this issue.

**Comment by Kukkimonsuta** [comment]
I'll try to get to this asap, however it's likely not just about adding the new version as there was some legacy context stuff removed and needs some testing.

**Comment by Kukkimonsuta** [comment]
I have published new version 1.2.0 https://www.npmjs.com/package/inversify-react which includes react 19, in the end the legacy context issue was just in tests. Please let me know if you find any issues.

**Comment by dbelob** [comment]
> I have published new version 1.2.0 https://www.npmjs.com/package/inversify-react which includes react 19, in the end the legacy context issue was just in tests. Please let me know if you find any issues.

@Kukkimonsuta Thank you very much!!! Excellent library!

**Comment by beverlylaurel** [comment]
@Kukkimonsuta gracias 👌
